lessons/Early-Math/associative-property/langs/en-us/pages/which-circles-make-the-math-easier.adoc

```diff
@@ -0,0 +1,67 @@
+= Which Circle Makes the Computation Easier?
+
+++++
+<style>
+div.circleevalsexp { width: auto; }
+</style>
+++++
+
+Put a check mark by the Circle of Evaluation which applies the Associative Property to make computation simpler. Then, evaluate the Circle.
+
+[.FillVerticalSpace,cols=".^1a,^.^6a,^.^8a,^.^8a,^.^3a", stripes="none", options="header"]
+|===
+|	 | Numeric Expression | Option 1 | Option 2 | Evaluate
```

Comment:
We've defined Arithmetic Expression in our glossary - want to stick to that instead of Numeric Expression?